controllers/root_controller.go

```diff
@@ -38,3 +41,21 @@ func (u *Root) Index(c web.C, w http.ResponseWriter, r *http.Request) {
 	}
 	tpl.ExecuteWriter(pongo2.Context{"title": "Fascia"}, w)
 }
+
+func (u *Root) About(c web.C, w http.ResponseWriter, r *http.Request) {
```

Comment:
__[golint]__
* [golint] exported method Root.About should have comment or be unexported